packages/clerk-js/src/ui/common/RemoveResourceForm.tsx

```diff
@@ -25,10 +25,16 @@ export const RemoveResourceForm = withCardStateProvider((props: RemoveFormProps)
   };
 
   return (
-    <FormContainer headerTitle={title}>
+    <FormContainer
+      headerTitle={title}
+      headerSubtitle={messageLine1}
+    >
       <Form.Root onSubmit={handleSubmit}>
-        <Text localizationKey={messageLine1} />
-        <Text localizationKey={messageLine2} />
+        {/*<Text localizationKey={messageLine1} />*/}
```

Comment:
Don't we need this anymore? We should remove completely if that's the case.

Reply:
Yes, this is a leftover. I will remove it. I used the key above.